pyre/pyre-interpreter/src/module/_abc/app_abc.py

```
@@ -0,0 +1,42 @@
+# `app_abc.py:15-44 SimpleWeakSet`.  The registry and both caches are
+# instances of this, so `_get_dump` can hand out the `data` sets and a
+# collected entry drops itself through the callback the set installs.
+#
+# Held at app level rather than rebuilt over the raw set primitives because
+# the callback closes over a weakref to the set: the discard has to run with
+# the set still reachable but no longer keeping itself alive through it.
+from _weakref import ref
+
+
+class SimpleWeakSet:
+    def __init__(self, data=None):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)